powerdns/models/requests.py

```diff
@@ -25,6 +25,36 @@
 log = logging.getLogger(__name__)
 
 
+def can_auto_accept_record(request, user, action):
```

Comment:
Maybe better name for 'request' is: 'record_request' because it confuses me with django request